packages/eslint-plugin-pf-codemods/src/rules/helpers/renamePropsOnNode.ts

```diff
@@ -1,33 +1,66 @@
-import { Rule } from 'eslint';
+import { Rule } from "eslint";
 import {
   JSXOpeningElement,
   ImportSpecifier,
   JSXAttribute,
   JSXIdentifier,
-} from 'estree-jsx';
-import { renameSinglePropOnNode, Renames } from './renameSinglePropOnNode';
+  ImportDefaultSpecifier,
+  ImportDeclaration,
+} from "estree-jsx";
+import { renameSinglePropOnNode, Renames } from "./renameSinglePropOnNode";
+
+interface ImportDefaultSpecifierWithParent extends ImportDefaultSpecifier {
```

Comment:
Not sure if we'd ever re-use it, but may not hurt to move this into the interfaces helper file